web/src/connectors/categories-modal-connector/utils.ts

```diff
@@ -39,11 +40,11 @@ export const getDefaultValues = (category: Category | undefined) => {
     } = category || {};
 
     return {
-        categoryId: '',
+        categoryId: id || '',
         name: name ?? '',
         parentId: parent,
         type: type ?? 'box',
-        withParentLabel: !parent,
+        withParentLabel: !!parent,
```

Comment:
Why did we update this condition here?

Reply:
The way it was:
- When creating a category, parent checkbox was automatically selected by default. So, if you didnt want it to have a parent, you needed to manually uncheck the checkbox.
- When updating a category without parent, it was automatically selected to have a parent as well, so you needed to manually uncheck the checkbox

This way
- On category creation, checkbox starts unselected
- When updating a category that does not have parent, checkbox starts unchecked
- When updating a category that has a parent, checkbox comes checked and the previously selected parent comes selected